frappe/public/js/frappe/list/list_view.js

```diff
@@ -508,7 +508,11 @@ frappe.views.ListView = class ListView extends frappe.views.BaseList {
 	}
 
 	get_form_link(doc) {
-		return '#Form/' + this.doctype + '/' + doc.name;
+		const docname = doc.name.match(/[%'"]/)
```

Comment:
Why not always? Also, couldn't there be more special chars? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent